### ISSUE TITLE
Update to new RN version for build patch

### ISSUE THREAD

**Issue by maximilize** [react-native-tvos/react-native-tvos#448]
### Description

Seems like RN created a bug which affects the build process of many older versions. See https://github.com/facebook/react-native/issues/35210

We should apply this patch to solve such build errors:

```
diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index 2e9d8e5..2ccc9e7 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -464,3 +464,18 @@ afterEvaluate {
         }
     }
 }
+
+// Patch needed for https://github.com/facebook/react-native/issues/35210
+// This is a patch to short-circuit the "+" dependencies inside the
+// users' app/build.gradle file and the various .gradle files of libraries.
+// As using plain "+" dependencies causes Gradle to always download the latest,
+// this logic forces Gradle to use latest release in the minor series.
+project.rootProject.allprojects {
+    configurations.all {
+        resolutionStrategy {
+            force "com.facebook.react:react-native:0.69.+"
+            force "com.facebook.react:hermes-engine:0.69.+"
+        }
+    }
+}
+
```

### Version

0.69.6-1

### Output of `npx react-native info`

System:
    OS: Linux 6.0
    CPU: (20) x64 12th Gen Intel(R) Core(TM) i7-1280P
    Memory: 20.54 GB / 31.07 GB
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.12.1 - ~/.local/node-v18.12.1-linux-x64/bin/node
    Yarn: 1.22.19 - ~/.local/node-v18.12.1-linux-x64/bin/yarn
    npm: 8.19.2 - ~/.local/node-v18.12.1-linux-x64/bin/npm
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: 1.8.0_352 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0 
    react-native: 0.69.5 => 0.69.5 
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

Create a new app with tvos and build it.

**Comment by yusufyildirim** [comment]
This will be solved with the upcoming 0.69.6-2 release.

**Comment by douglowder** [comment]
@yusufyildirim is correct. @maximilize this is already in the 0.69.6 release branch as https://github.com/react-native-tvos/react-native-tvos/commit/9fe8815c087445b9297a5438aa1d64f2c7e5e691
